nilearn/reporting/_get_clusters_table.py

```diff
@@ -267,7 +272,19 @@ def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
                            Rows corresponding to subpeaks will not have a value
                            in this column.
         ================== ====================================================
+        
+    label_maps : :obj:`list`
+        Returned if return_label_maps=True
+        List of Niimg-like objects of cluster label maps.
+        If two_sided==True, first and second maps correspond to positive and negative tails.
     """
+    # Add future warning message
+    future_warn_message = (
+    "The 'return_label_maps' parameter will default to 'True' in a couple of releases."
+    "The 'return_label_maps' argument will be removed a couple of releases later."
```

Comment:
```suggestion
        "and will be removed in release `0.14.0`."
```

Reply:
We may want to get the table and not the image. I don't think we should remove the parameter in release 0.14.